src/components/TodoList.jsx

```diff
@@ -74,7 +74,7 @@ function TodoList({ item, deleteFn, id, handleChange }) {
             alignItems="center"
           >
             <Grid item xs={8}>
-              <ListItem data-testid="list-item" key={id} paddingLeft={10}>
+              <ListItem data-testid="list-item" key={id} paddingleft={10}>
```

Comment:
It should be `paddingLeft`

Reply:
for paddingLeft it gives warning in the console that "React does not recognize the `paddingLeft` prop on a DOM element" and suggests to use in lowercase

Reply:
Which means `ListItem` may not accept paddingLeft. You need to apply it to the `Grid`.